app/src/main/res/layout/activity_rider_view_request.xml

```diff
@@ -120,6 +120,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_marginTop="5dp"
+            android:id="@+id/imageView_requestStatus"
```

Comment:
Yaa. Don't know if it's just because the other stuff in the list adapter is taking up too much space or what.